int32 id
float32 confidence
geometry_msgs/Pose2D center
geometry_msgs/Point32 leftTop
geometry_msgs/Point32 bottomRight
